index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+    <!-- Navigation Header -->
+
+    <header class="header">
+        <nav class="navbar">
+            <h4 class="header-name">Augustine Abah</h4>
```

Comment:
- [ ] Please to use the anchor `a` tag for your logo when designing a page. This is to ensure that it has a link back to the home page on multi-pages.